### PR TITLE
Crossbow

### PR DESCRIPTION
**Pull request by jbox144**
## About The Pull Request

This PR ports the crossbow from Skyrat.

Horny jokes aside, this makeshift weapon is a powerful utility for a prospecting greytider. It's crafted using many easy-to-aquire components and fires dirt cheap iron rods. The crossbow has to be charged however, up to 3 times, and each charge requires the user stay still for about half a second. A full charge crossbow fires its bolt to deal 30 damage and embeds the rod, but as stated, reloading can be cumbersome in-combat.

+ Cheap ammunition
+ High, once off damage
+ Embed chance
- Slow to reload
- Bulky, can only be carried in belt slot

Further balance changes may be needed, but I really do like this weapon, and hope I can even expand on it in the future. Additions such as using power cells to heat the rods to deal additional burn damage, or a repeater variant which is faster to fire but does less damage.

## Why It's Good For The Game

The crossbow a nice weapon which differs from conventional basaltic weapons and a reasonable thing you'd expect a crazy space man to assemble themselves.

## Changelog
:cl: BobJoga, Useroth
add: Adds the Crossbow, a makeshift weapon that fires iron rods.
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
I'd recommend reading our contributor guidelines, you need to credit the original author of the PR. Add their name after the first :cl: Link the original PR and co-commit if possible :)

File: austation/code/modules/projectiles/guns/ballistic/crossbow.dm [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+	. = ..()
+	var/bowstring = "The bow string is "
+	if (charge > 2)
+		bowstring = bowstring + "drawn back fully"
+	else if (charge > 1)
+		bowstring = bowstring + "drawn back most the way"
+	else if (charge > 0)
+		bowstring = bowstring + "drawn back a little"
+	else
+		bowstring = bowstring + "not drawn"
```
Comment (TheFakeElon) [inline austation/code/modules/projectiles/guns/ballistic/crossbow.dm:116]: use a switch statement

Comment (jupyterkat) [reply austation/code/modules/projectiles/guns/ballistic/crossbow.dm:116]: ah yes, the yandere else if

Comment (jbox144) [reply austation/code/modules/projectiles/guns/ballistic/crossbow.dm:116]: I can't locate the original PR but I can locate the contributors, PR description updated

Comment (jbox144) [reply austation/code/modules/projectiles/guns/ballistic/crossbow.dm:116]: wrong spot

**Comment by jbox144** [comment]
> I'd recommend reading our contributor guidelines, you need to credit the original author of the PR. Add their name after the first 🆑 Link the original PR and co-commit if possible :)
 I can't locate the original PR but I can locate the code contributors. Description updated.

File: austation/code/modules/projectiles/guns/ballistic/crossbow.dm (hunk 1 above)
Comment (jbox144) [reply austation/code/modules/projectiles/guns/ballistic/crossbow.dm:116]: To be fair this isn't my code

Comment (ZombiesVsPlants) [reply austation/code/modules/projectiles/guns/ballistic/crossbow.dm:116]: based

**Comment by TheFakeElon** [comment]
put the names on the same line as the CL, not under

**Comment by jbox144** [comment]
> 
> 
> put the names on the same line as the CL, not under

Sorry about being incompetent heh

**Comment by TheFakeElon** [comment]
Pfft, you're fine

**Comment by jupyterkat** [comment]
Half life 2 heated bolt crossbow when

**Comment by jbox144** [comment]
> 
> 
> Half life 2 heated bolt crossbow when

I plan on adding that as a modification latter, adding additional burn damage if a power cell is installed.

**Comment by MCterra10** [comment]
I updated your CL, you don't need to mention them lol.

**Comment by Kalvinalvaro** [comment]
Carry 6 crossbows in BoH and gamer them valids

**Comment by ZombiesVsPlants** [comment]
b-b-b-bASED

**Pull request merged**